### DISCUSSION TITLE
有关屏蔽视频的功能

### DISCUSSION THREAD

**Discussion by akeatcaihong** [BewlyBewly/BewlyBewly#776] [Ideas]
大大能添加按标题关键词屏蔽视频的功能吗？或者兼容篡改猴上已有的屏蔽插件，最近添加了（Bilibili 按标签、标题、时长、UP主屏蔽视频）脚本，但并不能生效。这是脚本信息https://github.com/tjxwork/bilibili_blocked_videos_by_tags。谢谢大佬

**Comment by hakadao** [comment]
有考慮過的，現在我們目前是先穩定下來處理效能問題，後續才敢加新功能

**Reply by akeatcaihong** [reply]
好的，谢谢大佬，希望越搞越好

**Comment by hakadao** [comment]
之後會有
https://github.com/BewlyBewly/BewlyBewly/pull/965
